make-rules/ips.mk

```diff
@@ -62,6 +62,10 @@ GENERATE_TRANSFORMS +=		$(WS_TOP)/transforms/generate-cleanup
 PKGMOGRIFY_TRANSFORMS +=	$(WS_TOP)/transforms/libtool-drop
 PKGMOGRIFY_TRANSFORMS +=	$(WS_TOP)/transforms/ignore-libs
 
+ifneq ($(GCC_ROOT), /usr/gcc/4.9)
```

Comment:
What I meant with the definition of a USERLAND_GCC_ROOT is that there are two variables needed:
1) the userland gcc used as system compiler
2) the component gcc with overriden GCC_ROOT
That's what you do by comparing if GCC_ROOT == /usrr/gcc/4.9; in this case USERLAND_GCC_ROOT=/usr/gcc/4.9.
My suggestion was to avoid hardcoding /usr/gcc/4.9 and put it in a variable, so that we avoid changing the hardcoded value in different locations whenever we migrate to a new compiler. @pyhalov @xen0l

Reply:
Do I understand correctly that you want to set USERLAND_GCC_ROOT to /usr/gcc/4.9? When and why will we have to change it?

Reply:
It should match the host system compiler. Maybe the name is not correct.

Reply:
still do not understand :(